src/Ursa/Controls/IPv4Box/IPv4Box.cs

```diff
@@ -283,11 +283,13 @@ protected override void OnPointerPressed(PointerPressedEventArgs e)
             {
                 if (e.ClickCount == 1)
                 {
-                    _imClient.SetPresenter(presenter);
-                    presenter.ShowCaret();
+                    _imClient.ShowInputPanel();
```

Comment:
`ShowInputPanel` is going to be removed in 12.0
But there will be another alternative for 11.3.x
https://github.com/AvaloniaUI/Avalonia/pull/19225